wishlist page/wishlist.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
+    <link rel="stylesheet" href="/homepage/homepage.css">
+    <link rel="stylesheet" href="/product listing page/products.css">
+    <link rel="stylesheet" href="wishlist.css"> <link rel="stylesheet" href="../css/base.css">
+    <link rel="stylesheet" href="../css/spacing.css">
+    <link rel="stylesheet" href="https://galaxia-designs.netlify.app/styles.css" />
+    <script src="https://kit.fontawesome.com/7f4b9645a7.js" crossorigin="anonymous"></script>
+</head>
+
+<body class="m-0">
+    <!-- header -->
+    <header>
+        <div class="container m-a ">
+            <nav class="nav-bar ">
+                <div class="left-side ">
+                    <a href="">MiS<span class="span-color">tO</span></a>
+                </div>
+                <div class="nav-link-container  ">
+
+                    <a class=" nav-link padding " href="">Women</a>
+                    <a class="nav-link padding" href="">Men</a>
+                    <a class="nav-link padding " href="">Kids</a>
+                </div>
+                <div class="search-bar m-5">
+                    <form action="#">
+
+                        <input type="search" placeholder=" Search..." name="search" class="icon-search"></form>
+                </div>
+                <div class="right-links">
+                    <ul>
+                        <li><a href=""><i class="fa-solid fa-heart wish-list padding-i color"></i></a>
+                        </li>
+                        <li><a href=""><i class="fa-solid fa-cart-shopping padding-i"></i></a>
+                        </li>
+                        <li <a href="" class="log-in padding-i">LOGIN</a></li>
```

Comment:
If these icons are not linked to any pages as of yet, then give it a placeholder `#`